src/api-docs/source/includes/_node_rpc_names.md

```diff
@@ -486,6 +486,69 @@ Name | Default |  Description
 --------- | --------- | --------- | -----------
 claim | Required | raw serialized hex-string
 
+## getdnssecproof
+
+```shell--cli
+hsd-rpc getdnssecproof <domain-name>
+```
+
+```shell--curl
+domain_name=foo.bar
```

Comment:
Recommend using camel case here, or just calling it `name`. You might also have to put the value in double-quotes so it works in cURL and JS methods.

Then in the examples below, be sure to use the same!
Test out the doc by straight up copy-pasting the whole blob into terminal, all three methods. See some of the surrounding docs for syntax i.e. usually declare the vars in a "let" at the top, so it will be visible in all three "modes".

Reply:
Verified that each of the commands work exactly as in the docs and have the let declared at the top for each example.